packages/api/tests/rich-text-detection.test.ts

```diff
@@ -319,9 +327,14 @@ describe('detectFacets', () => {
           },
         ],
       ],
+      ['this #​ should not be a tag', [], []],
+      ['this #​a should not be a tag', [], []],
+      ['this #a​b should be a tag', ['a'], [{ byteStart: 5, byteEnd: 7 }]],
+      ['no match \\u20e2: #⃢tag', [], []],
+      ['no match a\\u20e2b: #a⃢b', ['a'], [{ byteStart: 19, byteEnd: 21 }]],
```

Comment:
These actually contain a [combining enclosed screen](https://symbl.cc/en/20E2/#:~:text=Symbol%20Meaning,-Combining%20Enclosing%20Screen&text=The%20symbol%20%E2%80%9CCombining%20Enclosing%20Screen,U%2B20E2) as Mary pointed out